feature/floormap/src/main/res/layout/fragment_floormap.xml

```diff
@@ -19,11 +18,11 @@
             android:src="@drawable/ic_floor_map"
             android:paddingStart="16dp"
             android:paddingEnd="16dp"
+            android:tint="?attr/colorPrimary"
```

Comment:
Accessibility Scanner may suggest a better color. Did you already check it? Please let me know then!

Reply:
hmm.. seems ok?
![82970881_723546138173833_4361847014904823808_n](https://user-images.githubusercontent.com/13504295/73060774-8e9e9300-3ed3-11ea-8a0e-22b941cb587f.png)

Reply:
ColorOnBackground might be a better choice, but will result in the map being black under day theme, is that acceptable?

Reply:
The current change looks great! Thanks for your dedicated work.